packages/sku/scripts/postinstall.js

```diff
@@ -3,15 +3,26 @@
 import { readFile } from 'node:fs/promises';
 import { join } from 'node:path';
 import debug from 'debug';
-import chalk from 'chalk';
+import { styleText } from 'node:util';
```

Comment:
Aiming to slowly replace `chalk` usage over time. It's technically an unstable API given our current node target, but it's stable in current LTS versions and there aren't any API changes that affect our usage of it.